design-proposals/external-monitoring.md

```diff
@@ -0,0 +1,173 @@
+# Overview
+
+Recently there is a lot of ongoing effort to establish a monitoring solution for the Kubevirt organization.
+This effort is very important as a good monitoring solution & design directly impacts an operator's quality.
+For example, operator framework define the [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/),
+as a metric of an operator's maturity. Looking at the current state of Kubevirt, arguably we fill the requirements
+of the first 3 levels, which are "Basic install", "Seamless Upgrades" and "Full Lifecycle" management. That said,
+we probably have some more work to do in order to become level 4 and 5, which are "Deep insights" and "Auto Pilot".
```

Comment:
@iholder-redhat  KubeVirt is already in Deep Insights level and working to reach "Auto Pilot" level

Reply:
IIUC deep insights means that we have many metrics and alerts that provide insightful details of the cluster.
I thought this is a work in progress. Do you think we're already there?

Reply:
Sure, I'll change that.